reduxNews/Feature/News/ViewModel/NewsViewModel.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  ViewModel.swift
+//  reduxNews
+//
+//  Created by Владимир Бондарь on 10/1/18.
+//  Copyright © 2018 VladymyrBondar. All rights reserved.
+//
+
+import Foundation
+
+import Result
+import ReactiveCocoa
+import ReactiveSwift
+
+extension News {
+  final class ViewModel {
+    struct Inputs {
+      let viewWillAppear: Signal<Void, NoError>
+      let searchText: Signal<String?, NoError>
+    }
+    
+    struct Outputs {
+      let props: Property<NewsView.Props>
+    }
+    
+    init(){}
+    
+    func makeOutputs(inputs: Inputs) -> Outputs {
+      
```

Comment:
in case when function declaration is on 1 line don't put any newlines between signature and function body.